lib/kubeclient/common.rb

```diff
@@ -190,8 +190,8 @@ def define_entity_methods
         end
 
         # get a single entity of a specific type by name
-        define_singleton_method("get_#{entity.method_names[0]}") do |name, namespace = nil|
-          get_entity(klass, entity.resource_name, name, namespace)
+        define_singleton_method("get_#{entity.method_names[0]}") do |name, ns = nil, opts = {}|
```

Comment:
@abonas `rubocop` was yelling because the line was too long.  Open to a better way of handling this, but this was the best that I came up with.